packages/manager/modules/pci/src/projects/project/notebooks/dashboard/attach-data/attach-data.html

```diff
@@ -55,14 +61,61 @@
                 <oui-clipboard model="$row.mountPath"></oui-clipboard>
             </oui-datagrid-column>
             <oui-datagrid-column
-                title=":: 'pci_notebooks_tab_attach_data_page_volumes_column_permission' | translate"
+                title=":: 'pci_projects_project_training_jobs_submit_data_permission' | translate"
+                sortable="true"
+                property="permission"
+            >
+                <span
+                    data-ng-bind=":: 'pci_projects_project_training_jobs_submit_data_permission_' + $row.permission | translate"
+                ></span>
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                title=":: 'pci_projects_project_training_jobs_submit_data_cache' | translate"
+                sortable="true"
+            >
+                <span data-ng-bind="$row.cache"></span>
+            </oui-datagrid-column>
+        </oui-datagrid>
+
+        <!--Title and description-->
+        <div class="oui-box oui-box_light mb-4 p-0 border-0">
```

Comment:
**question:** Is there any reason to remove both padding and border on the `oui-box` element?

Element should looks like this:
https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-box-native--light

Reply:
There is a point indeed, with border and padding we do not get the expected header we want

Reply:
As discussed together, might be great to get rid of this `oui-box` since it contains only an `h4` element.